gradle-plugins/compose/src/test/test-projects/application/webJsWasm/composeApp/build.gradle.kts

```diff
@@ -0,0 +1,56 @@
+import org.jetbrains.kotlin.gradle.ExperimentalWasmDsl
+import org.jetbrains.kotlin.gradle.targets.js.webpack.KotlinWebpackConfig
+
+plugins {
+    id("org.jetbrains.kotlin.multiplatform")
+    id("org.jetbrains.compose")
+    id("org.jetbrains.kotlin.plugin.compose")
+}
+
+kotlin {
+    js(IR) {
+        outputModuleName.set("composeApp")
```

Comment:
If we want to test a custom name here, then you need a different name, because "composeApp" is a name of the module already